l10n - added Czech translation into xml

p-bo opened the pull request:
Added short and long description in Czech language, as well as keywords.

p-bo committed:
l10n - added Czech translation into xml

Added short and long description in Czech language, as well as keywords.

amma35 commented on financialreports.xml:
```diff
@@ -6,10 +6,12 @@
     <logo>https://raw.githubusercontent.com/InfotelGLPI/financialreports/master/financialreports.png</logo>
     <description>
         <short>
+            <cs><![CDATA[Finanční hlášení: Majektová situace. Tento zásuvný modul umožňuje vytvořit finanční výkaz (majektovou situaci) pro dané datum.<br />- Export do CSV, PDF]]></cs>
```
Can you add <cs> after <en>, please?

amma35 commented on financialreports.xml:
```diff
@@ -6,10 +6,12 @@
             <en><![CDATA[Financial report : Asset situation. This plugin allows you to generate a financial report (asset situation) for a given date.<br />- Export Csv, Pdf]]></en>
         </short>
         <long>
+            <cs><![CDATA[Finanční hlášení: Majektová situace. Tento zásuvný modul umožňuje vytvořit finanční výkaz (majektovou situaci) pro dané datum.<br /> - V nastavení zásuvného modulu přidáte identifikátor pro každý typ inventáře vybavení související s inventárním číslem (příklad: vaše notebooky mají neopakující se inventární číslo začínající na „NTB“ takže vyplníte nastavení modulu s NTB, s ...
```
Can you add <cs> after <en>, please?

p-bo committed:
Tweaked financialreports.xml

Ordered alphabetically for correct sorting when displayed on plugins.glpi-project.org
Revised list of languages to present only translations, that are at least from 50% done on Transifex (to don't confuse users about true availability in their languages).
Minor tweaks in Czech translation.

p-bo committed:
Tweaked financialreports.xml

Ordered alphabetically for correct sorting when displayed on plugins.glpi-project.org
Revised list of languages to present only translations, that are at least from 50% done on Transifex (to don't confuse users about true availability in their languages).
Minor tweaks in Czech translation.

p-bo reviewed (commented):
Ordered alphabetically for correct sorting when displayed on plugins.glpi-project.org
Revised list of languages to present only translations, that are at least from 50% done on Transifex (to don't confuse users about true availability in their languages).
Minor tweaks in Czech translation.
